sklearn/cluster/spectral.py

```diff
@@ -452,3 +452,88 @@ def fit(self, X):
     @property
     def _pairwise(self):
         return self.affinity == "precomputed"
+
+
+def spectral_clustering_path(affinity, n_clusters=range(2, 4),
+                             eigen_solver=None, random_state=None,
+                             n_init=10, n_jobs=1,
+                             eigen_tol=0.0, assign_labels='kmeans'):
+    """Apply clustering with successive values of k to a projection to the
```

Comment:
Please use the name `n_clusters` instead of `k`. Already it seems `k` is incorrectly used in the `spectral_clustering` documentation.